src/actions_eval.ts

```diff
@@ -1,17 +1,11 @@
-/* eslint-disable @typescript-eslint/ban-ts-comment */
-/* eslint-disable @typescript-eslint/no-unused-vars */
-
-import * as _vscode from "vscode";
+import _vscode from "vscode";
 
 import { createLogger } from "./logger";
 
-// @ts-ignore
+const vscode = _vscode;
```

Comment:
`(void)foo` is enough to tell tools (or at least typescript compiler) that `foo` is used.  Maybe worth a try?
```suggestion
const vscode = _vscode;
(void)vscode;
```

Reply:
Perfect. I wasn't sure of the best way to mark a variable as used in JavaScript and this is much better